### ISSUE TITLE
Reproducing GALAXY

### ISSUE THREAD

**Issue by richlee123** [richlee123/GALAXY#1]
I am trying to reproduce the pre-train checkpoint you shared on Github. For somehow, I could not get the same pre-training checkpoint. So I several questions came to my mind:
Q1: Stopping criteria for choosing pre-training & fine-tuning checkpoints. It seems to me that the stopping criteria is not based on validation loss. What was the criteria for choosing the final epoch number? For example, you select epoch 14 for pre-training and epoch 7 for MultiWOZ2.0.
Q2: The number of pre-training data. The UniDA dataset you shared on Github has 463,039, but this seems smaller than the sum of the training sets in eight datasets used for UniDA. Did you get the same checkpoint with the pre-train data you currently released?
Q3: GPU machines used for pre-training. It would be great if you can share what GPU machines you used to pre-train the GALAXY model. Perhaps that might be one of the reasons why I do not get the same result.
Thanks!
